server/mqtt.go

```diff
@@ -1990,45 +1965,41 @@ func (as *mqttAccountSessionManager) processJSAPIReplies(_ *subscription, pc *cl
 //
 // Run from various go routines (JS consumer, etc..).
 // No lock held on entry.
-func (as *mqttAccountSessionManager) processRetainedMsg(_ *subscription, c *client, _ *Account, subject, reply string, rmsg []byte) {
+func (as *mqttAccountSessionManager) processRetainedMsg(sub *subscription, c *client, acc *Account, subject, reply string, rmsg []byte) {
```

Comment:
`sub` and `acc` are not needed. The change was done during dev but I forgot to revert I guess. I will update as part of other requested changes that may come as part of the review.